### ISSUE TITLE
train rt-detr ended up with half of the mAP50 of the others

### ISSUE THREAD

**Issue by whx1116** [ultralytics/ultralytics#16463]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I trained RT-DETR on visdrone dataset in ultralytics for 150epochs, with the same parameters (SGD,lr0.01, etc.) except for batchsize4 (8 in other papers), and I ended up with half of the mAP50 of the others (after reading a couple of papers the mAP50 is focused on 40- 45%, my training result is only about 20%), tried rtdetr-l and resnet18 both.

**Comment by UltralyticsAssistant** [comment]
👋 Hello @whx1116, thank you for reaching out to Ultralytics 🚀!

We appreciate your detailed information on training RT-DETR with the VisDrone dataset. For further diagnostics, we recommend visiting our [Docs](https://docs.ultralytics.com) where you'll find helpful resources, including [Training Tips](https://docs.ultralytics.com/guides/model-training-tips/) and [Examples](https://docs.ultralytics.com/usage/python/). These may offer insights into optimizing your training process.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us investigate further.

In the meantime, ensure that your environment is up to date. You can upgrade everything by running:

```bash
pip install -U ultralytics
```

For real-time support, join our [Discord](https://ultralytics.com/discord) 🎧. For in-depth discussions, visit [Discourse](https://community.ultralytics.com), or connect with our community on [Subreddit](https://reddit.com/r/ultralytics).

## Environments

Run YOLOv8 in verified environments with all dependencies pre-installed:
- **Notebooks**: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
- **Google Cloud**: [GCP Quickstart](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon**: [AWS Quickstart](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker**: [Docker Quickstart](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>
Check the badge above for current CI test status. Our system tests all YOLOv8 modes and tasks on multiple platforms daily.

This is an automated response, and an Ultralytics engineer will address your query soon. Thanks for your patience! 😊

**Comment by Skillnoob** [comment]
Can you share the training code?
